### ISSUE TITLE
problem with gunicorn logs when debug is enabled and mflog loaded

### ISSUE THREAD

**Issue by thefab** [metwork-framework/mflog#9]
```
2019-04-11T16:34:44.988636Z    [ERROR] (gunicorn.error#28290) Traceback (most recent call last):
  File "/home/fab/metwork/mfserv/build/opt/python3/lib/python3.5/site-packages/gunicorn/glogging.py", line 353, in access
    self.access_log.info(self.cfg.access_log_format, safe_atoms)
  File "/home/fab/metwork/mfext/build/opt/python3_core/lib/python3.5/logging/__init__.py", line 1285, in info
    self._log(INFO, msg, args, **kwargs)
  File "/home/fab/metwork/mfext/build/opt/python3_core/lib/python3.5/logging/__init__.py", line 1421, in _log
    self.handle(record)
  File "/home/fab/metwork/mfext/build/opt/python3_core/lib/python3.5/logging/__init__.py", line 1431, in handle
    self.callHandlers(record)
  File "/home/fab/metwork/mfext/build/opt/python3_core/lib/python3.5/logging/__init__.py", line 1493, in callHandlers
    hdlr.handle(record)
  File "/home/fab/metwork/mfext/build/opt/python3_core/lib/python3.5/logging/__init__.py", line 861, in handle
    self.emit(record)
  File "/home/fab/metwork/mfext/build/opt/python3/lib/python3.5/site-packages/mflog/__init__.py", line 48, in emit
    logger.info(record.msg, *(record.args), **kwargs)
  File "/home/fab/metwork/mfext/build/opt/python3/lib/python3.5/site-packages/structlog/stdlib.py", line 69, in info
    return self._proxy_to_logger("info", event, *args, **kw)
  File "/home/fab/metwork/mfext/build/opt/python3/lib/python3.5/site-packages/structlog/stdlib.py", line 119, in _proxy_to_logger
    method_name, event=event, **event_kw
  File "/home/fab/metwork/mfext/build/opt/python3/lib/python3.5/site-packages/structlog/_base.py", line 191, in _proxy_to_logger
    args, kw = self._process_event(method_name, event, event_kw)
  File "/home/fab/metwork/mfext/build/opt/python3/lib/python3.5/site-packages/structlog/_base.py", line 151, in _process_event
    event_dict = proc(self._logger, method_name, event_dict)
  File "/home/fab/metwork/mfext/build/opt/python3/lib/python3.5/site-packages/structlog/stdlib.py", line 280, in __call__
    event_dict["event"] = event_dict["event"] % args
TypeError: format requires a mapping
```

**Comment by thefab** [comment]
maybe a struclog issue :

```
self.cfg.access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
safe_atoms: {'{script_name}e': '', '{x-real-ip}i': '127.0.0.1', 't': '[11/Apr/2019:18:38:13 +0200]', '{x-forwarded-path}i': '/fab', '{accept-language}i': 'fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3', 'q': '', '{http_cookie}e': 'csrftoken=HIDDEN; grafana_session=dff02494af3715abccf8a5e759d08c3c', '{cookie}i': 'csrftoken=HIDDEN; grafana_session=dff02494af3715abccf8a5e759d08c3c', '{path_info}e': '/fab', '{x-forwarded-proto}i': 'http', '{x-forwarded-for}i': '127.0.0.1', '{wsgi.run_once}e': False, '{wsgi.multiprocess}e': True, 'f': '-', '{wsgi.multithread}e': False, '{http_x_real_ip}e': '127.0.0.1', 'H': 'HTTP/1.1', '{server_name}e': 'localhost', '{wsgi.version}e': (1, 0), '{gunicorn.socket}e': <socket.socket fd=9, family=AddressFamily.AF_UNIX, type=SocketKind.SOCK_STREAM, proto=0, laddr=/home/fab/metwork/mfserv/var/gunicorn_app_fab_main.socket>, 'b': '12', '{http_x_forwarded_for}e': '127.0.0.1', '{connection}i': 'close', '{host}i': 'localhost:18868', '{http_accept_language}e': 'fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3', 'l': '-', 'a': 'Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0', '{raw_uri}e': '/fab', '{wsgi.url_scheme}e': 'http', '{x-request-id}i': 'e585c596a6225c0b12bff13a0ef507bb', '{http_upgrade_insecure_requests}e': '1', '{server_software}e': 'gunicorn/19.9.0', '{upgrade-insecure-requests}i': '1', '{user-agent}i': 'Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0', 'D': 2336, 'U': '/fab', 'B': 12, '{http_accept}e': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', '{http_accept_encoding}e': 'gzip, deflate', '{wsgi.file_wrapper}e': <class 'gunicorn.http.wsgi.FileWrapper'>, '{http_x_forwarded_path}e': '/fab', '{request_method}e': 'GET', '{http_connection}e': 'close', '{server_port}e': '18868', 'L': '0.002336', 'p': '<12207>', '{wsgi.errors}e': <gunicorn.http.wsgi.WSGIErrorsWrapper object at 0x7f95c812eba8>, '{wsgi.input}e': <gunicorn.http.body.Body object at 0x7f95c812ebe0>, '{http_host}e': 'localhost:18868', 'h': '', '{http_x_request_id}e': 'e585c596a6225c0b12bff13a0ef507bb', 'u': '-', 's': '200', 'r': 'GET /fab HTTP/1.1', '{content-length}o': '12', '{accept}i': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'T': 0, '{accept-encoding}i': 'gzip, deflate', '{http_user_agent}e': 'Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0', '{server_protocol}e': 'HTTP/1.1', '{remote_addr}e': '', 'm': 'GET', '{query_string}e': '', '{http_x_forwarded_proto}e': 'http', '{content-type}o': 'text/plain'}
```

**Comment by thefab** [comment]
fixed by d52fea8f765c2037348a5e865b8b5ac3f7e42076
